warehouse/templates/base.html

```diff
@@ -246,7 +246,7 @@
             <h2>Help</h2>
           </li>
           <li><a href="https://packaging.python.org/installing/">Installing packages</a></li>
-          <li><a href="https://packaging.python.org/distributing/">Uploading packages</a></li>
+          <li><a href="https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives">Uploading packages</a></li>
```

Comment:
Let's just link to the entire tutorial instead

```suggestion
          <li><a href="https://packaging.python.org/tutorials/packaging-projects/">Uploading packages</a></li>
```